bors.toml

```diff
@@ -0,0 +1,6 @@
+status = [
+    "proj ubuntu",
```

Comment:
I'm like 60% sure this bors config is correct, but can't test it until this is merged into your pr.